src/index.js

```diff
@@ -94,29 +94,33 @@ const _extends = Object.assign || function (target) {
 
 
 export const getTargets = (targets = {}) => {
-  const targetOps = _extends({}, targets);
+  const targetOpts = _extends({}, targets);
 
-  if (targetOps.node === true || targetOps.node === "current") {
-    targetOps.node = getCurrentNodeVersion();
+  if (targetOpts.node === true || targetOpts.node === "current") {
+    targetOpts.node = getCurrentNodeVersion();
+  }
+
+  if (targetOpts.hasOwnProperty("uglify") && !targetOpts.uglify) {
```

Comment:
It will allow `uglify: null` and use it exactly like a `true`.  I think we must check other targets the same way.